README.md

```diff
@@ -118,13 +118,13 @@ directory = "./data"
 ```
 
 #### Root `fossil` config block
-| Option             | Default                    | Description                                            |
-| ------------------ | -------------------------- | ------------------------------------------------------ |
-| `fossil.port`      | 8001                       | Port fossil server listens on                          |
-| `fossil.prom-port` | 2112                       | Port fossil server servers `/metrics` on               |
-| `fossil.verbose`   | 0                          | Configures the log level [0: info, 1: debug, 2: trace] |
-| `fossil.host`      | `"fossil://local/default"` | Connection string client will connect to               |
-| `fossil.local`     | true                       | Configures output logs to be in plaintext              |
+| Option             | Default       | Description                                            |
+| ------------------ |---------------| ------------------------------------------------------ |
+| `fossil.port`      | 8001          | Port fossil server listens on                          |
+| `fossil.prom-port` | 2112          | Port fossil server servers `/metrics` on               |
+| `fossil.verbose`   | 0             | Configures the log level [0: info, 1: debug, 2: trace] |
+| `fossil.host`      | `"./default"` | Connection string client will connect to               |
```

Comment:
Should this be ?
```suggestion
| `fossil.host`      | `"file://./default"` | Connection string client will connect to               |
```

Reply:
Either will work using the new stuff -- they should be equivalent